client/src/pages/ResourcesPage/ResourcesBar.jsx

```diff
@@ -48,6 +53,17 @@ function ResourcesBar(props) {
         debounceFn();
     }, [title, tags, sortBy]);
 
+    const getPersonalPosts = async () => {
```

Comment:
nit: if you wanted to refactor this function, i think a better/cleaner structure could be:
```
getPersonalPosts (inPersonal) => {
response = inPersonal ? API.getPersonalPosts : API.getPosts;
if (response == ok) {
setInPersonal(!inPersonal);
props.postsSetter(response.posts);
}
}
```